rubocop.yml

```diff
@@ -336,6 +336,41 @@ Naming/HeredocDelimiterCase:
 Naming/HeredocDelimiterNaming:
   Enabled: false
 
+Naming/InclusiveLanguage:
+  Enabled: true
+  CheckIdentifiers: true
+  CheckConstants: true
+  CheckVariables: true
+  # CheckStrings is false in the rubocop defaults
+  CheckStrings: true
+  CheckSymbols: true
+  CheckComments: true
+  CheckFilepaths: true
```

Comment:
IMO this new cop (which helps us enforce the rule introduced by #167, by the way) makes sense for naming in variables, constants, etc, but when it comes to the contents of strings it doesn't make much sense to me. I think we should omit this configs and stick with RuboCop's defaults.

Reply:
My counterargument for including strings is that it catches references in error messages ("add xyz to whitelist to allow ...") and test descriptions. Test descriptions often have references to the same identifiers, constants, variable where we'd look for offenses.